LocalSettings.php

```diff
@@ -4110,6 +4110,13 @@
 		'wmgUseTranslate' => false,
 		'wmgUseUniversalLanguageSelector' => false,
 	],
+	'wgULSIMEEnabled' => [
+		'default' => true,
+		'gratispaideiawiki' => false,
+	],
+	'wgULSWebfontsEnabled' => [
```

Comment:
setting the default only again...

Reply:
This isn't the default value: https://github.com/wikimedia/mediawiki-extensions-UniversalLanguageSelector/blob/master/extension.json#L101

Reply:
Done